org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/WListbox.java

```diff
@@ -1255,5 +1277,100 @@ public void setAllowIDColumnForReadWrite(boolean allowIDColumnForReadWrite)
 	{
 		this.allowIDColumnForReadWrite = allowIDColumnForReadWrite;
 	}
+	
+	   /**
+     * Retrieve the width of the column
+     * 
+     * @param columnName the name of the column to find the width of.
+     * @param columnList list of the columns and their width
+     * @return the width of the column
+     */
+	private String getColumnWidth(String columnName, List<String>columnList) 
+	{
+		String width ="null";
+		String[] w;
+		
+		for (String column : columnList)
+		{
+			w = column.trim().split("=");
+			if (w[0].equals(columnName))
+			{
+				return w[1];
+			}
+		}
+		return width;
```

Comment:
should be simpler to just return null here and Util.isEmpty(...) below will works for both null and empty string.

Reply:
Util.isEmpty(...) return 0 or 1. The null value is save in the custom field in the database and is used to show that we haven't been able to get or save any column width.